bool  max_current_reached
float64  commanded_effort
float64  curr_time
float64  current_vel
